web/src/lib/domain-logic/getNavBarBusinessTitle.ts

```diff
@@ -27,13 +23,6 @@ export const getNavBarBusinessTitle = (
     foreignBusinessTypeIds,
   } = business.profileData;
 
-  const isRemoteWorkerOrSeller = (): boolean => {
```

Comment:
[sand] So I'm totally down for making this domain logic, but we have like six other places where we locally declare an isRemoteWorkerOrSeller function which should also be consolidated in this PR or as a follow up

Reply:
https://www.pivotaltracker.com/story/show/186749997